examples/assets_dynamic_partitions/assets_dynamic_partitions/assets.py

```diff
@@ -0,0 +1,125 @@
+import json
+import os
+import urllib.request
+import zipfile
+
+import pendulum
+import requests
+from dagster import DynamicPartitionsDefinition, Output, asset, get_dagster_logger
+from pandas import DataFrame
+
+releases_partitions_def = DynamicPartitionsDefinition(name="releases")
+
+
+@asset(
+    partitions_def=releases_partitions_def,
+    io_manager_key="warehouse",
+    metadata={"partition_expr": "release_tag"},
+)
+def releases_metadata(context) -> DataFrame:
+    """Table with a single row per release"""
+    release_tag = context.partition_key
+
+    response = requests.get(
+        f"https://api.github.com/repos/dagster-io/dagster/releases/tags/{release_tag}",
+        auth=("sryza", os.environ["GITHUB_API_KEY"]),
```

Comment:
Maybe move the username to an env var too?

Reply:
oops good call